app/services/middleware_topology_service.rb

```diff
@@ -1,30 +1,27 @@
 class MiddlewareTopologyService < TopologyService
+  include UiServiceMixin
```

Comment:
Not just yet, but in the future it will, once the MW icons are coming from UiServiceMixin. I intend to do a commit with such change shortly, on this same PR.